### PR TITLE
refactor and incorporate google api

### PR DESCRIPTION
**Pull request by harrisdan10**

### CODE REVIEW

**Review by Jimmvu — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Monoton&display=swap" rel="stylesheet"> 
     <!-- Internal CSS -->
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./styles/style.css">
```
Comment (Dr3adnought) [inline index.html:15]: updated to reflect additional folder

File: index.html [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h1>The College Fridge</h1>
     <!-- Bootstrap js -->
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
     <!-- Internal js -->
-    <script src="./edamam.js"></script>
+    <script type="module" src="./scripts/script.js"></script>
```
Comment (Dr3adnought) [inline index.html:51]: updated to reflect additional folder and name change of file

File: styles/style.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .bg{
-    background-image: url(./sf-images/fridge.jpg);
+    background-image: url(../sf-images/fridge.jpg);
```
Comment (Dr3adnought) [inline styles/style.css:2]: updated to reflect additional folder

**Review by Dr3adnought — APPROVED** [review]

**Comment by Dr3adnought** [comment]
updated

**Pull request closed without merging**